lib/cisco_node_utils/platform.rb

```diff
@@ -164,7 +164,7 @@ def self.virtual_services
       # convert to expected format
       virts_hsh = {}
       virts.each do |serv|
-        # rubocop:disable Style/AlignHash, Style/ExtraSpacing
+        # rubocop:disable Style/AlignHash
```

Comment:
There was a bug previously in rubocop that required us to disable both rules (otherwise Rubocop would get into a recursive loop of trying to satisfy two mutually conflicting rules). That bug should be fixed in 0.35 - if you enable both rules, does this pass now?